packages/plugins/plugin-assistant/src/capabilities/blueprint-definition.ts

```diff
@@ -19,7 +19,11 @@ export default () => {
         key: 'dxos.org/blueprint/assistant',
         name: 'Assistant',
         instructions: templates.system,
-        tools: functions.map((tool) => ToolId.make(tool.name)),
+        tools: [
+          ...functions.map((tool) => ToolId.make(tool.name)),
+          // TODO(wittjosiah): Factor out.
+          ToolId.make('show'),
```

Comment:
Can we import ShowToolkit and get the tool by name `ShowToolkit.tools.show.name` (or somthing like this

Reply:
this is fair but if we're going to do that cross-plugin we need to separate the implementation so that plugins and not pull in the full implementation when importing. Do you have suggestions for how that should be setup?